README.md

```diff
@@ -1,2 +1,8 @@
-# scci-core-services-api
-API definitions of the SCCI Platform core services
+# rokwire-core-services-api
+API definitions of the Rokwire Platform core services
+
+## How to view API design doc 
+You can view API design and document on Swaggerhub.
+
+- Sign up a free account at [SwaggerHub](https://swagger.io/tools/swaggerhub/). 
+- Log into your account, "Create New" API by importing rokwire.yaml file  
```

Comment:
Could you please also add instructions to use Docker for running a Swagger UI instance with our YAML file. This will help us to deploy our API definitions later. You can refer to Swagger documentation at https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/installation.md

Reply:
Added